src/azure/accountService.ts

```diff
@@ -77,9 +77,13 @@ export class AccountService {
         return account;
     }
 
-    public async createSecurityTokenMapping(): Promise<Map<string, string>> {
-        let mapping = new Map<string, string>();
-        mapping.set(this.getHomeTenant(this.account).id, await this.refreshToken(this.account));
+    public async createSecurityTokenMapping(): Promise<any> {
+        // let mapping = new Map<string, string>();
+        let mapping = {};
+        mapping[this.getHomeTenant(this.account).id] = {
+            token: await this.refreshToken(this.account)
+        };
+        // mapping.set(this.getHomeTenant(this.account).id, await this.refreshToken(this.account));
```

Comment:
Please remove the commented out code
Perhaps add a `@TODO` for a task in future to match the type for mapping in `vscode-mssql` and `sqltoolsservice`